<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePackMgrWidgetClass</class>
 <widget class="QWidget" name="FilePackMgrWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilePackMgrWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="group_box_vs_version">
       <property name="title">
        <string>VS version</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="TableViewWidget" name="widget_vs_version" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>result_output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="OutputResultWidget" name="widget_ret_output" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_box_exe_dll">
      <property name="title">
       <string>Exe And DLL Version</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="TableViewWidget" name="widget_exe_dll" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="group_box_vdproj">
      <property name="title">
       <string>VDProj Version</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="TableViewWidget" name="widget_vdproj" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TableViewWidget</class>
   <extends>QWidget</extends>
   <header>tableviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OutputResultWidget</class>
   <extends>QWidget</extends>
   <header>outputresultwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
